demo/project_demo00-SecOps/simulator/src/main.rs

```diff
@@ -1,16 +1,20 @@
-use csv::WriterBuilder as CsvWriterBuilder;
 use futures::executor::block_on;
 use mockd::datetime::date_range;
 use rand::{random, thread_rng, Rng};
 use rdkafka::{
     admin::{AdminClient, AdminOptions, NewTopic, TopicReplication},
-    client::DefaultClientContext,
+    // client::DefaultClientContext,
     config::{FromClientConfig, RDKafkaLogLevel},
     producer::{BaseRecord, DefaultProducerContext, Producer, ThreadedProducer},
     util::Timeout,
     ClientConfig,
 };
 use serde::Serialize;
+use serde_json::json;
+use std::{
+    sync::Arc,
+};
```

Comment:
I think that rustfmt would condense this to `use std::sync::Arc`. Perhaps rustfmt has not been run.